app/controllers/reviews_controller.rb

```diff
@@ -27,6 +28,21 @@ def destroy
     # code
   end
 
+  def edit
+    @review = Review.find(params[:review_id])
+  end
+
+  def update
+    @shelter = Shelter.find(params[:shelter_id])
+    review = Review.find(params[:review_id])
+    if review.update(review_params)
```

Comment:
I noticed you did the flash[:notice] when it's a success so I could change this later on as well so that it's in sync.